packages/@aws-cdk/aws-ec2/lib/instance.ts

```diff
@@ -245,6 +245,13 @@ export interface InstanceProps {
    * @default - false
    */
   readonly requireImdsv2?: boolean;
+
+  /**
+   * Whether "Detailed Monitoring" is enabled for this instance
```

Comment:
Can you add a comment here as well about the extra cost (same thing you have in the readme)

Reply:
@corymhall added a note about the extra charges here as well.